styles.css

```diff
@@ -0,0 +1,463 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Roboto:ital,wght@1,100;1,400&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Lato, sans-serif;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.semi-nav {
+  display: none;
+}
+
+.nav-bar {
+  display: none;
+}
+
+.overlay {
+  background-color: #40312a;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 50%;
+  bottom: 0;
+  right: 0;
+  display: none;
+}
+
+.main-menu {
+  display: flex;
+  flex-direction: column;
+  padding-top: 106px;
+  padding-left: 50px;
+  gap: 20px;
+}
+
+.menu-item {
+  color: #fff;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 40px;
+}
+
+.cancel {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  position: fixed;
+  right: 2%;
+  padding: 8px;
+}
+
+.hide {
+  display: flex;
+  flex-direction: row;
+  gap: 30%;
+}
+
+.bars img {
+  margin: 15px;
+}
+
+.header-body {
+  background-image: url(Images/bg-01.jpg);
+  width: 100%;
+  height: 90vh;
+}
+
+.header-text {
+  display: flex;
+  flex-direction: column;
+  color: #ec5242;
+  padding-top: 50px;
+}
+
+.header-text > * {
+  margin-bottom: 1px;
+}
+
+.header-text h1 {
+  font-size: 2rem;
+  font-weight: 900;
+  margin-top: 10px;
+}
+
+.header-text h2 {
+  font-weight: 400;
+}
+
+.summary {
+  text-align: center;
+  padding: 2%;
+  background-color: #f6f3f3;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(1px);
+  border: 1px solid wheat;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.summary h2 {
+  font-size: 20px;
+  font-weight: 400;
+  color: #a9962b;
+}
+
+.summary p {
+  text-align: center;
+  font-weight: 400;
+  font-size: 19px;
+}
+
+/* mid-part section */
+
+.mid-part {
+  background-color: white;
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.mid-logo {
+  align-self: center;
+  padding: 2%;
+}
+
+.mid-heading {
+  text-align: center;
+  color: #a9962b;
+}
+
+.activity {
+  align-self: center;
+  border-bottom: 1px solid #a9962b;
+}
+
+.activity h1 {
+  font-size: 20px;
+  font-weight: 600;
+  color: #a9962b;
+}
+
+.activity-para p {
+  text-align: center;
+  padding: 6%;
+  font-weight: 500;
+  font-size: 18px;
+}
+
+.mid-img {
+  display: flex;
+  flex-direction: column;
+  margin: 5%;
+  margin-bottom: 1px;
+  width: 90%;
+}
+
+.mid-img1 {
+  width: fit-content;
+  height: 45vh;
+}
+
+.mid-img1 img {
+  width: 95%;
+}
+
+.mid-img2 {
+  width: fit-content;
+  height: 45vh;
+}
+
+.mid-img2 img {
+  width: 100%;
+}
+
```

Comment:
- [ ] I can see that you made great job on this project but there still something you have to work on so that your project will look awesome , as we are imitating this [Cindy shin design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and if we compare your design they are somehow different, your image should be the background which comes after your news description, this images below will explain more kindly check the different :point_down: 

Figma :point_down: 
![windowofnews](https://user-images.githubusercontent.com/88894525/198232381-13634567-4a07-4f5a-868a-f4bf9254c5f1.png)

Your Design 
![foryour](https://user-images.githubusercontent.com/88894525/198232850-d3dcdebd-ad42-4e1f-9aa6-874c273e54a8.png)

Reply:
Hello @Alfredbis29 I made the required changes. Please check I only have 1 review left.